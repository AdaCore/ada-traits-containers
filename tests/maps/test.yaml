title: 'maps'
description: 'Basic test for maps'
driver: 'build_and_exec'
