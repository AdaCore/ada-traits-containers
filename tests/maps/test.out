Value for one is  1
Value for four is  4
Value for seven is  7
